README.md

```diff
@@ -18,6 +18,9 @@ The component has some additional configuration options:
 | Option         | Type     | Default value | Description |
 |--------------|-----------|------------|------------|
 | productid | string  | undefined | Your app ID in the Microsoft Store. You can find this value by going to your URL in the Microsoft Web Store and grabbing the last part of the URL. For OneNote, for example, the web link is https://www.microsoft.com/en-us/p/onenote-for-windows-10/9wzdncrfhvjl, so OneNote's product ID is `9wzdncrfhvjl` |
+| cid | string | undefined | Your app campaign code for the Microsoft Store. |
+| popup | "true" or "" | "true" | Configure the badge to open your app in MiniPDP mode or window mode respectively. |
+| darkMode | "true" or "" or "auto" | "true" | Configure the badge theme to be in dark mode, light mode, or auto mode respectively. |
```

Comment:
Can you make sure this is rendering right? When I view the preview in Github, it looks like cid, popup, and darkmode are all in the same row:

![image](https://user-images.githubusercontent.com/312936/170375087-20c73af4-e4a7-475c-8af0-78203d0c18f3.png)